ci/disallow_c_comments_in_migrations.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+set -euo pipefail
+
+# We do not use c-style comments in migration files as the stripped
+# zero-length migration files cause warning during packaging
+# See #3115 for more info
+
+# In this file, we aim to keep the indentation intact by capturing whitespaces,
+# and reusing them if needed. GNU sed unfortunately does not support lookaround assertions.
+
+# /* -> --
+find src/backend/distributed/sql/*.sql -print0 | xargs -0 sed -i 's#/\*#--#g'
+
+# */ -> `` (empty string)
+# remove all whitespaces immediately before the match
+find src/backend/distributed/sql/*.sql -print0 | xargs -0 sed -i 's#\s*\*/\s*##g'
```

Comment:
```suggestion
find src/backend/distributed/sql/*.sql -print0 | xargs -0 sed -i 's#^\s*\*/\s*##g'
```

Reply:
```
/* this is a valid comment */
```
and I should be able to remove `*/` even if we have non-space characters in the same line.

Reply:
Ah that's fair